src/ui/mixins/context-menu-gesture.js

```diff
@@ -15,23 +17,29 @@ export default {
   },
   mounted() {
     //TODO: touch support
-    this.$el.addEventListener('contextmenu', this.showContextMenu);
+    this.$nextTick(() => {
```

Comment:
Why do we need this?

Reply:
Vue 3 changes the way refs work. We need the nextTick to ensure it's available.

Reply:
Vue 3 changes the way refs work. We need the nextTick to ensure it's available.